### ISSUE TITLE
Options for different GAN model encodings in Lode Runner

### ISSUE THREAD

**Issue by schrum2** [schrum2/MM-NEAT#483]
I believe we will want to eventually allow different types of Lode Runner GAN encodings for comparison. Add a parameter: lodeRunnerDistinguishesSolidAndDiggableGround. The default should be false, which means that all ground is diggable, and that both B and b from VGLC map to 3.

However, if lodeRunnerDistinguishesSolidAndDiggableGround is true, then the solid ground B should map to 6.

Then, from now onward, every model you train should somehow indicate in the filename (preferably in a parsable way) which of these encodings was used. The reason for this is that we ultimately want to be able to use the SetGANModel option in the interactive interface to load models of either type, and have them display appropriately.

In fact, once these encoding options are available, make sure that a model of each type exists in the repo (even if it doesn't work well ... we just need something to experiment with).

**Comment by kdsteckel** [comment]
Pushed a commit earlier, tagged with #394  accidentally, that added models that we talked about yesterday. 
LodeRunnerEpochFirstFiveOneGround_10000_20_6.pth is the model that combines all the ground tiles into diggable ground and does not have a spawn point.
LodeRunnerEpochFirstFiveAllGround_10000_20_7.pth is the model that is from the first five levels and distiguishes between diggable and solid ground, still excludes spawn point.
@schrum2
